distributed/bokeh/scheduler.py

```diff
@@ -306,7 +306,9 @@ def __init__(self, scheduler, width=600, **kwargs):
             self.nbytes_figure = nbytes
 
             processing.y_range = nbytes.y_range
-            self.root = row(nbytes, processing, sizing_mode='scale_width')
+            # self.root = row(nbytes, processing, sizing_mode='scale_width', name='current_load')
+            self.root_1 = nbytes
```

Comment:
We do have the benefit of setting multiple roots for things that share datasources/update callbacks.